Using the v2 container runtime. For more details see: https://discuss.circleci.com/t/docker-executor-infrastructure-upgrade/52282
task-agent version 1.0.256686-185c83c7
Downloading task-agent: success after 403.228726ms.
System information:
 Server Version: 5.1.2
 Storage Driver: overlay
  Backing Filesystem: xfs
 Cgroup Driver: systemd
 Cgroup Version: 2
 Kernel Version: 6.8.0-1018-aws
 Operating System: ubuntu
 OSType: linux
 Architecture: amd64

Starting container k8s.gcr.io/pause:3.6
  image is cached as k8s.gcr.io/pause:3.6, but refreshing...

[1A[2Kfbe1a72f5dcd: Already exists [1B
[1A[2K6270bb605e12: Pulling fs layer [1B[1A[2K6270bb605e12: Download complete [1B[1A[2K6270bb605e12: Download complete [1BStarting container cimg/python:3.9
Warning: No authentication provided, using CircleCI credentials for pulls from Docker Hub.
  image cache not found on this host, downloading docker.io/cimg/python:3.9

[1A[2K31c5345d5902: Already exists [1B
[1A[2K9228d7d64782: Already exists [1B
[1A[2K3b6608a99f95: Already exists [1B
[1A[2K0a27abb621ec: Already exists [1B
[1A[2Kb7e51fe35101: Already exists [1B
[1A[2K4f4fb700ef54: Already exists [1B
[1A[2K737a6d414a94: Already exists [1B
[1A[2K171ec96ceb16: Already exists [1B
[1A[2K0cfd23a39181: Already exists [1B
[1A[2K589c1e9d083e: Already exists [1B
[1A[2K03dbce9c1d80: Already exists [1B
[1A[2K01007420e9b0: Already exists [1B
[1A[2Kaafd43b90acb: Pulling fs layer [1B[1A[2Kaafd43b90acb: Download complete [1B[1A[2Kaafd43b90acb: Download complete [1Bcimg/python:3.9:
  using image docker.io/cimg/python@sha256:d2241875cee76c59b40d1efd3a6d7945c35de0013d50e1e7ab6afe407a3781f4
  pull stats: Image was already available so the image was not pulled
  time to create container: 39ms
Time to upload agent: 413.058221ms
Time to start containers: 282.663722ms
